webpay/pin/tests/test_forms.py

```diff
@@ -5,6 +5,7 @@
 from nose.tools import eq_
 
 from lib.solitude.api import client
+from webpay.base import dev_messages as msg
 from webpay.pin import forms
```

Comment:
I'm not seeing any tests for `ResetPinForm`. I guess they are all handled by the view tests. Perhaps they should move here and the view can just test success/fail?